app/services/change_schedule.rb

```diff
@@ -80,6 +81,18 @@ def cohort
 
 private
 
+  def can_change_cohort_leaving_billable_declarations?
+    return false unless participant_profile.ecf?
+    return false unless attempt_to_change_cohort_leaving_billable_declarations
+    return true if participant_profile.eligible_to_change_cohort_and_continue_training?(cohort:)
+
+    participant_profile.eligible_to_change_cohort_back_to_their_payments_frozen_original?(cohort:)
+  end
+
+  def cohort_start_years_delta
```

Comment:
is this required?

Reply:
No, missed this, left over from previous implementation